RZBluetooth/Profiles/RZBDeviceInfo.m

```diff
@@ -35,17 +144,51 @@ + (CBCharacteristicProperties)characteristicPropertiesForKey:(NSString *)key
 
 + (id)valueForKey:(NSString *)key fromData:(NSData *)data
 {
-    return [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+    if ([key isEqualToString:@"systemId"]) {
+        return [RZBSystemId valueForKey:key fromData:data];
+    }
+    else
+    if ([key isEqualToString:@"pnpId"]) {
```

Comment:
Can you consolidate the `else` and `if`?  ie: `else if () {` -- Here and a few other places below.